5_Value.md

```diff
@@ -0,0 +1,26 @@
+# Metric: Value
+
+Activity Metric | Description
+--- | ---
+Development Cost | An estimate of the market cost to develop the functionality of the project.  See how [OpenHub does it](https://blog.openhub.net/project_codebase_cost/)

```

Comment:
I like the idea of estimating development cost, but I don't think COCOMO, which is basically what OpenHub uses, is very useful in the usual FOSS context. I would keep the metric, but skip the example.

Reply:
@jgbarah I'm not proposing any specific calculation (COCOMO or otherwise) - I was merely referring to prior art for this kind of metric.